src/main/java/com/epam/esm/config/SpringConfig.java

```diff
@@ -0,0 +1,67 @@
+package com.epam.esm.config;
+
+import org.springframework.context.annotation.*;
+import org.springframework.context.support.ReloadableResourceBundleMessageSource;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.springframework.web.servlet.LocaleResolver;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.servlet.i18n.CookieLocaleResolver;
+import org.springframework.web.servlet.i18n.LocaleChangeInterceptor;
+
+import javax.sql.DataSource;
+
+@Configuration
+@ComponentScan("com.epam.esm")
+@EnableWebMvc
+@EnableTransactionManagement
+public class SpringConfig implements WebMvcConfigurer {
+
+    private static final String ENCODING = "UTF-8";
+    private static final String SET_LOCALE = "classpath:locale/messages";
+    private static final String LOCALE = "locale";
+
+    @Bean
+    public PlatformTransactionManager transactionManager(DataSource dataSource) {
+        return new DataSourceTransactionManager(dataSource);
+    }
+
+    @Bean
+    public JdbcTemplate jdbcTemplate(DataSource dataSource) {
+        return new JdbcTemplate(dataSource);
+    }
+
+    @Bean
+    public ReloadableResourceBundleMessageSource reloadableResourceBundleMessageSource() {
+        ReloadableResourceBundleMessageSource reloadableResourceBundleMessageSource = new ReloadableResourceBundleMessageSource();
+
+        reloadableResourceBundleMessageSource.setBasename(SET_LOCALE);
+        reloadableResourceBundleMessageSource.setCacheSeconds(-1);
+        reloadableResourceBundleMessageSource.setUseCodeAsDefaultMessage(false);
+        reloadableResourceBundleMessageSource.setDefaultEncoding(ENCODING);
+
+        return reloadableResourceBundleMessageSource;
+    }
+
+    @Bean
+    public LocaleResolver localeResolver() {
+        return new CookieLocaleResolver();
+    }
+
+    @Bean
+    public LocaleChangeInterceptor localeChangeInterceptor() {
+        LocaleChangeInterceptor localeChangeInterceptor = new LocaleChangeInterceptor();
+        localeChangeInterceptor.setParamName(LOCALE);
```

Comment:
Please use the `Accept-Language` header to set locale. Not URL parameter.

Reply:
Fixe. Now we use "Accept-language" header instead